README.md

```diff
@@ -18,13 +18,13 @@ Installation and Use
 You can install this package directly from GitHub with pip:
 
 ```bash
-$ pip -e git+https://github.com/michaelbrooks/rq-win.git#egg=rq-win
+$ pip install git+https://github.com/michaelbrooks/rq-win.git#egg=rq-win
```

Comment:
Things work this way on my machine (but it's probably trivial for you to re-test). -e, as far as I understand is to install in development mode, retaining the cloned package dir somewhere under python/src, yes.